service-query/src/main/kotlin/org/veupathdb/service/mblast/query/sql/queries/select-query-config.kt

```diff
@@ -202,6 +204,8 @@ WHERE
     WHERE
       ${Column.UserID} = ?
   )
+ORDER BY
+  ${Column.Position}
```

Comment:
Could possibly use a comment on what this position refers to and why we order by it. Is this just to ensure sub-jobs are returned in a consistent order?